src/pages/home/index.js

```diff
@@ -44,14 +27,14 @@ export const Home = () => {
         </ContentTitle>
         <CategoryContainer />
       </div>
-
+ 
       <div class="top-products-container">
         <ContentTitle styles={"offer-header"}>
           MOST DAZZLING OFFERS
         </ContentTitle>
         <div class="product-container">
           {topProductsData.map(item => (
-            <ProductCard item={item} key={item._id} clickHandler={addToWishListHandler}/>
+            <ProductCard item={item} key={item._id} clickHandler={({item}) => addToWishListHandler({item})}/>
```

Comment:
for clickHandler function take into another function defined upwards don't make inline function definitions (a suggestion)